.changeset/early-seahorses-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Upgrade `miniflare` to `2.1.0`
```

Comment:
What would be really great is if this changeset linked to the changelog for the new version of miniflare.

Reply:
🚀